### DISCUSSION TITLE
exclude favicon.ico from other templates than favicon-detection.yaml

### DISCUSSION THREAD

**Discussion by savushkin-yauheni** [projectdiscovery/nuclei-templates#3509] [Q&A] [answered]
Hi team

Maybe any tech template that contains `favicon.ico` path should be moved to technologies/favicon-detection.yaml or at least be duplicated there?  What do you think ?

e.x:

https://github.com/projectdiscovery/nuclei-templates/blob/0c7d13d1528de5c34aa0059f4de87e005041b9c1/exposed-panels/sysaid-panel.yaml#L12

Thanks!

**Comment by olearycrew** [comment, marked as answer]
I think that is intentional, it is looking for other matchers in the responses from that so I would call that valid and placed correctly
